problem/src/main/java/org/zalando/problem/UnknownStatus.java

```diff
@@ -3,7 +3,7 @@
 import javax.annotation.concurrent.Immutable;
 
 @Immutable
-final class UnknownStatus implements StatusType {
+public final class UnknownStatus implements StatusType {
```

Comment:
`UnknownStatus` is an implementation detail and shouldn't be public within the `problem` main module.